PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.kt

```diff
@@ -89,7 +89,7 @@ class PayPalVaultRequest
 
         if (enablePayPalAppSwitch && !appLink.isNullOrEmpty() && !userAuthenticationEmail.isNullOrEmpty()) {
             parameters.put(ENABLE_APP_SWITCH_KEY, enablePayPalAppSwitch)
-            parameters.put(OS_VERSION_KEY, Build.VERSION.SDK_INT)
+            parameters.put(OS_VERSION_KEY, Build.VERSION.RELEASE)
```

Comment:
Does setting this to `.RELEASE` cover all future major and minor versions too?

Reply:
➕ 1 - I'm not sure what the difference between `SDK_INT` and `RELEASE` results in

Reply:
Previously, an integer was being sent instead of a string. Regarding your question, `SDK_INT` refers to the `API level`, while `RELEASE` refers to the `version number`, as shown in the table in this [link](https://en.wikipedia.org/wiki/Android_version_history). Do you think it's better to keep sending `SDK_INT` and just cast it to a string?

Reply:
I think casting the existing `SDK_INT` value might make things easier on the reporting end since the values will all be whole integers.

Separate question - is this value sent to the API for reporting purposes?

Reply:
Done: [97363bb](https://github.com/braintree/braintree_android/pull/1162/commits/97363bbf1dbf1258c6cbbbaae23524cc75326c83)

Reply:
Oh actually I just noticed we don't pass the SDK version here on iOS, we pass `UIDevice.current.systemVersion` - so want to triple check which is the correct value.

Reply:
> Separate question - is this value sent to the API for reporting purposes?

For now, that’s correct, it’s just for reporting purposes.

> Oh actually I just noticed we don't pass the SDK version here on iOS, we pass UIDevice.current.systemVersion - so want to triple check which is the correct value.

After confirming, since it’s only used for reporting purposes for now, we’ll send the `SDK API Level`. This will allow us to have more granularity in the future if needed.